src/content/docs/extensions/algos/scc.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Strongly Connected Components
+---
+
+A strongly connected component (SCC) in a directed graph is a maximal subgraph where every pair of
+vertices is mutually reachable - meaning for any two nodes `u` and `v` in the component, there exists a
+directed path from `u` to `v` and a directed path from `v` to `u`. 
+This mutual reachability property means that nodes within an SCC form a cycle where they can all
+reach each other through directed paths, and no additional nodes can be added to the component while
+maintaining this property.
+
+Kuzu implements two types of SCC algorithms:
+- Parallel BFS-based [coloring algorithm](https://www.cs.rpi.edu/~slotag/pub/SCC-IPDPS14.pdf)
+- DFS-based [Kosaraju's algorithm](https://en.wikipedia.org/wiki/Kosaraju%27s_algorithm)
+
+You can choose between either of the two algorithms, depending on your use case.
```

Comment:
We should say more here on the difference of these two implementations, and give recommendations on which to use under what kind of use cases.

Reply:
Actually we have explained this. Let me close this comment